app/src/main/java/ch/epfl/sdp/entity/PlayerManager.java

```diff
@@ -3,11 +3,7 @@
 import java.util.ArrayList;
 
 public class PlayerManager {
-    private static ArrayList<Player> players;
-
-    public PlayerManager() {
-        players = new ArrayList<>();
-    }
+    private static ArrayList<Player> players = new ArrayList<>();
```

Comment:
Cool, so we don't need to instantiate it randomly,

Reply:
Also nice for testing (because before we had to do it)

Reply:
Actually it's not great for testing, but it helps to keep the code clean.